app/src/main/java/com/example/campusguide/MapsActivity.kt

```diff
@@ -35,6 +38,7 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
         val mapFragment = supportFragmentManager
                 .findFragmentById(R.id.map) as SupportMapFragment
         mapFragment.getMapAsync(this)
+        ObjectBox.init(this.applicationContext)
```

Comment:
Should we move the ObjectBox stuff to the BuildingHighlights class?

Reply:
I'd keep it there in case someone else would need to access the database. This is required to "start" the database so that you can access it anywhere in the app.

Reply:
Good point!